msmbuilder/msm/msm.py

```diff
@@ -154,7 +156,7 @@ def fit(self, sequences, y=None):
         None will not be counted. The mapping_ attribute will not include the
         NaN or None.
         """
-        _ = self._setup(sequences)
+        _ = self._build_counts(sequences)
```

Comment:
might as well completely omit the `_ = `

(the reason why this is sometimes used is if you are unpacking a tuple and don't care about one of the values)
```python
five, _ = (5, SomeCrazyObject())
print(five)
```